### DISCUSSION TITLE
$call.checkErrorLevel returned with 1

### DISCUSSION THREAD

**Discussion by inunezfe** [NREL/ReEDS-2.0#132] [Q&A: Execution]
Hi ReEDS team, i need help with an issue running the model, the file a_writedata.gms gets stucked on this step, this image is from gamslog.txt

<img width="1470" alt="lfs " src="https://user-images.githubusercontent.com/111766958/206025516-8171e51b-6ec0-4c36-b3f1-3f610ed01bce.png">

Then i looked in lstfiles/1_inputs.lst and this is the information that i get 

<img width="1470" alt="image" src="https://user-images.githubusercontent.com/111766958/206025052-92f4f0d1-fd76-40bd-a6b9-0ec1f13855e4.png">

I cant seem to find the issue with that. Also I have reset the repo per [this thread](https://github.com/NREL/ReEDS_OpenAccess/discussions/96) and it still is not working.

I have not made any changes to the cases.csv file.

Thanks for your help!

**Comment by wesleyjcole** [comment]
Can you confirm that the repo was checked out using git lfs?  You can do this by lookin in inputs\variability\EFS_Load and making sure that the files there are large (~300 MB) rather than 1 KB pointer files.

If these large files look to be checked out correctly, can you run LDC_prep.py line by line to see where it fails?  E.g., does it load the packages properly?

**Reply by inunezfe** [reply]
I can cofirm the repo was checked out using git lfs

<img width="737" alt="image" src="https://user-images.githubusercontent.com/111766958/206210648-7c1c4e40-8684-44b3-9432-6ffce892d370.png">

This is what i get when i try to run LDC_prep.py, but i do not know how to interpret what is going on

<img width="1470" alt="debug1" src="https://user-images.githubusercontent.com/111766958/206210819-8d9e411e-ce0d-44dc-b1b3-19d0189f16e3.png">

**Reply by wesleyjcole** [reply]
Are you able to run LDCprep.py line-by-line to see where it throws an error?  For example, open the script in Spyder and walk through the lines to see where the error occurs.

You might also be able to run LDCprep.py from the command line as you are doing here, but you need to make sure you are passing in all the arguments from the gamslog.txt file.

**Reply by inunezfe** [reply]
I ran the LDCprep.py file making sure i was passing the filepath arguments for basedir and inputs case on the debug variables

<img width="748" alt="image" src="https://user-images.githubusercontent.com/111766958/206500230-fed7a11a-bb2e-42ae-a3a2-2ce5ea5e5dba.png">

and i got this error 

<img width="1468" alt="image" src="https://user-images.githubusercontent.com/111766958/206500479-403f92b3-e9f7-4080-9c65-81f102503072.png">

Where can i find that filepath?

**Reply by wesleyjcole** [reply]
Ahh, it looks like you are trying to run ReEDS with the "individual sites" capability turned on.  The data for the individual sites are not in the repository because the file sizes are so large (7 years of hourly profiles for 50,000 sites across the U.S.).  You should be able to change that switch to use one of the other settings, and then ReEDS should run fine.
